src/game/overlay.c

```diff
@@ -243,18 +244,21 @@ void Overlay_DrawHealthBar()
     int32_t show = g_HealthBarTimer > 0 || hit_points <= 0
         || g_Lara.gun_status == LGS_READY;
     switch (g_Config.healthbar_showing_mode) {
-    case T1M_BSM_ALWAYS:
-        show = 1;
-        break;
-    case T1M_BSM_NEVER:
-        show = 0;
+    case T1M_BSM_DEFAULT:
+        show;
```

Comment:
what is this suppose to be?

Reply:
The DEFAULT cases don't need any extra options like the other cases, but I wanted to represent that it is a case.

Reply:
so default doesn't need to modify show. 
which this "does" but also looks like 'somebody forgot to set the default value' at which point you could put a comment to explain, at which you could then just put the comment and not worry about odd "compiler doesn't mind" cases.

Reply:
I’ll just change it to only a break. Idk why I thought to leave show; lol. Late night brain fart.

Reply:
I should just remove that DEFAULT case and make a (C keyword) default case that just breaks.